plugin/trino-blackhole/src/test/java/io/trino/plugin/blackhole/TestBlackHoleSmoke.java

```diff
@@ -174,6 +175,22 @@ public void dataGenerationUsage()
         assertThatQueryDoesNotReturnValues("DROP TABLE nation");
     }
 
+    @Test
+    public void testMaterializedView()
+    {
+        String viewName = "test_materialized_view_" + randomNameSuffix();
+        queryRunner.execute("CREATE MATERIALIZED VIEW " + viewName + " AS SELECT * FROM tpch.tiny.nation");
+
+        try {
+            // reading
+            MaterializedResult rows = queryRunner.execute("SELECT * FROM " + viewName);
+            assertEquals(rows.getRowCount(), 25);
```

Comment:
Do we need to check for listing of MVs ?

Reply:
listing added